packages/task/package.json

```diff
@@ -0,0 +1,165 @@
+  "homepage": "https://lit.dev/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/lit/lit.git",
+    "directory": "packages/task"
+  },
+  "type": "module",
+  "main": "index.js",
+  "module": "index.js",
+  "typings": "index.d.ts",
+  "directories": {
+    "test": "test"
+  },
+  "exports": {
+    ".": {
+      "types": "./development/index.d.ts",
+      "development": "./development/index.js",
+      "default": "./index.js"
+    },
+    "./deep-equals.js": {
+      "types": "./development/deep-equals.d.ts",
+      "development": "./development/deep-equals.js",
+      "default": "./deep-equals.js"
+    },
+    "./task.js": {
+      "types": "./development/task.d.ts",
+      "development": "./development/task.js",
+      "default": "./task.js"
+    }
+  },
+  "files": [
+    "/development/",
+    "!/development/test/",
+    "/deep-equals.{d.ts,d.ts.map,js,js.map}",
+    "/index.{d.ts,d.ts.map,js,js.map}",
+    "/task.{d.ts,d.ts.map,js,js.map}"
+  ],
+  "scripts": {
+    "build": "wireit",
+    "build:ts": "wireit",
+    "build:ts:types": "wireit",
+    "build:rollup": "wireit",
+    "test": "wireit",
+    "test:dev": "wireit",
+    "test:prod": "wireit",
+    "checksize": "wireit"
+  },
+  "wireit": {
+    "build": {
+      "dependencies": [
+        "build:ts",
+        "build:ts:types",
+        "build:rollup",
+        "../lit:build",
+        "../reactive-element:build"
+      ]
+    },
+    "build:ts": {
+      "command": "tsc --build --pretty",
+      "dependencies": [
+        "../lit:build:ts:types",
+        "../reactive-element:build:ts:types"
+      ],
+      "clean": "if-file-deleted",
+      "files": [
+        "src/**/*.ts",
+        "tsconfig.json"
+      ],
+      "output": [
+        "development",
+        "tsconfig.tsbuildinfo"
+      ]
+    },
+    "build:ts:types": {
+      "command": "treemirror development . \"**/*.d.ts{,.map}\"",
+      "dependencies": [
+        "../internal-scripts:build",
+        "build:ts"
+      ],
+      "files": [],
+      "output": [
+        "*.d.ts{,.map}",
+        "test/**/*.d.ts{,.map}"
+      ]
+    },
+    "build:rollup": {
+      "command": "rollup -c",
+      "dependencies": [
+        "build:ts"
+      ],
+      "files": [
+        "rollup.config.js",
+        "../../rollup-common.js"
+      ],
+      "output": [
+        "deep-equals.js{,.map}",
+        "index.js{,.map}",
+        "task.js{,.map}",
+        "test/**/*.js{,.map}"
+      ]
+    },
+    "checksize": {
+      "command": "rollup -c --environment=CHECKSIZE",
+      "dependencies": [
+        "build:ts"
+      ],
+      "files": [
+        "rollup.config.js",
+        "../../rollup-common.js"
+      ],
+      "output": []
+    },
+    "test": {
+      "dependencies": [
+        "test:dev",
+        "test:prod"
+      ]
+    },
+    "test:dev": {
+      "command": "MODE=dev node ../tests/run-web-tests.js \"development/**/*_test.js\" --config ../tests/web-test-runner.config.js",
```

Comment:
Nit: `MODE=dev` and `MODE=prod` (in the next test) could be moved into wireit `env`, but non blocking.

Reply:
Done